bleak/backends/bluezdbus/manager.py

```diff
@@ -780,6 +780,20 @@ async def get_services(
 
         return services
 
+    def get_device_props(self, device_path: str) -> Device1:
```

Comment:
I would prefer to stick with having methods to get individual properties using the `_get_device_property()` so that we get a more helpful error message when things go wrong.